tasks/configure.yml

```diff
@@ -19,25 +19,40 @@
     chdir: "/home/{{ zookeeper.user }}/bin"
   register: aws_cluster_autodiscover
   until: aws_cluster_autodiscover | success
-  retries: 2
+  retries: 4
   delay: 15
   when: zookeeper.aws_cluster_autodiscover.enabled
 
+- name: Wait for server
+  pause:
+    seconds: "{{ zookeeper.wait_for_server }}"
+  when: zookeeper.aws_cluster_autodiscover.enabled
+
 - name: Set autodiscover id based AWS autodiscover script
   set_fact:
     zookeeper:
       aws_cluster_autodiscover:
         data: "{{ aws_cluster_autodiscover.stdout | from_json }}"
   when: zookeeper.aws_cluster_autodiscover.enabled
 
+- name: Setup Zookeeper environment config
+  become: yes
+  become_user: "{{ zookeeper.user }}"
```

Comment:
can you please use 
```
template:
  owner: "{{ zookeeper.user }}"
```
... and remove `become_user`

This will stop working in next Ansible release - http://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user